contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -846,9 +846,7 @@ var EditMetadataEditor = BaseViews.BaseView.extend({
       data: this.get_intl_data()
     }));
     var self = this;
-    var tags = _.reject(State.contenttags.pluck("tag_name"), function(tag){
-      return self.shared_data.shared_tags.indexOf(tag) >= 0;
-    });
+    var tags = State.Store.getters.contentTags.map(tag => tag.tag_name).filter(tagName => self.shared_data.shared_tags.indexOf(tagName) < 0);
```

Comment:
The contentTag getter has a pretty simple implementation now (could achieve the same by accessing `State.Store.state.contentTag`.

Could possibly simplify the two lines that use it if the contentTag getter "plucks" `tag_name` for you and returns an array of strings.

Reply:
Thanks for review, @jonboiser - yeah, I'll check out all the places it actually gets accessed and see if having a getter that does more work under the hood might help.